data.py

```diff
@@ -19,7 +19,7 @@ class Data:
 
 
     def __init__(self):
-        self.redis = redis.Redis("localhost")
+        self.redis = redis.StrictRedis(host="localhost", port=6379, charset="utf-8", decode_responses=True)
```

Comment:
I am not sure why do you need to Strict define the Redis connection, it is hardcoded.